content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,210 @@
+   enable specific tasks.
+
+*  **Portable:** A configuration bundle for a complex containerized workload
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. User accounts don't exist in the Kubernetes API
+server; instead, the API server treats user identities as opaque data. You can
+authenticate as a user account using multiple methods.
+
+{{< table caption="Comparison between service accounts and users" >}}
+
+| Description | ServiceAccount | User or group |
+| --- | --- | --- |
+| Location | Kubernetes API (ServiceAccount object) | External |
+| Scope | Namespace | Cluster |
+| Access control | Kubernetes RBAC or other [authorization mechanisms](/docs/reference/access-authn-authz/authorization/#authorization-modules) | Kubernetes RBAC or other identity and access management mechanisms |
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts {#default-service-accounts}
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions other than the
+[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
+that Kubernetes grants to all authenticated accounts. If you delete the default
+ServiceAccount object in a namespace, the {{<glossary_tooltip text="control plane" term_id="control-plane">}}
+replaces it with a new one.
+
+If you deploy a Pod in a namespace, and you don't manually assign a
+ServiceAccount object to the Pod, Kubernetes assigns the `default` service
+account for that namespace to the Pod. You can change this behavior by setting
+the `automountServiceAccountToken` field in your Pod specification to `false`.
+
+## Use cases for Kubernetes service accounts {#use-cases}
+
+As a general guideline, you can use service accounts to provide identities in
+the following scenarios:
+
+* Your Pods need to communicate with the Kubernetes API server, for example in
+  situations such as the following:
+  *  Providing read-only access to sensitive information stored in Secrets.
+  *  Granting [cross-namespace access](TODO), such as allowing a Pod in namespace
+     `example` to read, list, and watch for Lease objects in the
+     `kube-node-lease` namespace.
+* Your Pods need to communicate with an external service. For example,
+  [Authenticating to a private image registry using an `imagePullSecret`](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
+* An external service needs to communicate with the Kubernetes API server. For
+  example, authenticating to the cluster as part of a CI/CD pipeline.
+
```

Comment:
_optional_
```suggestion
* You use third-party security software in your Kubernetes cluster. The third-party software
  relies on the ServiceAccount identity of different Pods to group those Pods into different
  contexts.

```
[example](https://projectcalico.docs.tigera.io/security/service-accounts)

Reply:
Done (didnt link to example)

Reply:
Ah, the example was just for context, not to link to. Thanks. All good here.